config/routes.rb

```diff
@@ -42,7 +42,7 @@
 
   put "resources/:id/onesky", to: "resources#push_to_onesky"
 
-  get "/translations/files/:path",
+  get "/#{Package::TRANSLATION_FILES_PATH}:path",
```

Comment:
I would actually argue that `/translations/files/` here is distinct from the constant for S3.